.swiftlint.yml

```diff
@@ -1,3 +1,3 @@
+swiftlint_version: 0.54.0
 parent_config: https://raw.githubusercontent.com/Automattic/swiftlint-config/0f8ab6388bd8d15a04391825ab125f80cfb90704/.swiftlint.yml
 remote_timeout: 10.0
```

Comment:
I was going to suggest to move `swiftlint_version` to the remote config, but then it would not be available for CocoaPods and the rest of the automation to read 👌